### PR TITLE
Merge MySql enhancement from generation to master

### PR DESCRIPTION
**Pull request by msJinLei**
## Description

https://github.com/Azure/azure-powershell/pull/13717
https://github.com/Azure/azure-powershell/pull/13838

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/MySql/custom/Get-AzMySqlConnectionString.ps1 [hunk 1]
```diff
@@ -34,7 +34,7 @@ function Get-AzMySqlConnectionString {
         [System.String]
         ${SubscriptionId},
 
-        [Parameter(ParameterSetName='GetViaIdentity', Mandatory, ValueFromPipeline, HelpMessage = 'The source server object to create replica from.')]
+        [Parameter(ParameterSetName='GetViaIdentity', Mandatory, ValueFromPipeline, HelpMessage = 'Name of the server')]
```
Comment (wyunchi-ms) [inline src/MySql/custom/Get-AzMySqlConnectionString.ps1:37]: Help message need to modify.

File: src/MySql/exports/Get-AzMySqlConnectionString.ps1 [hunk 2]
```diff
@@ -36,9 +36,8 @@ COMPLEX PARAMETER PROPERTIES
 
 To create the parameters described below, construct a hash table containing the appropriate properties. For information on hash tables, run Get-Help about_Hash_Tables.
 
-INPUTOBJECT <IServer>: The source server object to create replica from.
+INPUTOBJECT <IServer>: Name of the server
```
Comment (wyunchi-ms) [inline src/MySql/exports/Get-AzMySqlConnectionString.ps1:39]: Maybe here also need to modify.

File: src/MySql/exports/New-AzMySqlFlexibleServer.ps1 [hunk 3]
```diff
@@ -15,23 +15,95 @@
 
 <#
 .Synopsis
-Creates a new MySQL flexible server
```
Comment (wyunchi-ms) [inline src/MySql/exports/New-AzMySqlFlexibleServer.ps1:18]: Why here need to change?

File: src/MySql/utils/nouns.txt [hunk 4]
```diff
@@ -0,0 +1,138 @@
+abalone
```
Comment (wyunchi-ms) [inline src/MySql/utils/nouns.txt:1]: It maybe better to move to another place.

File: src/MySql/custom/Get-AzMySqlConnectionString.ps1 (hunk 1 above)
Comment (msJinLei) [reply src/MySql/custom/Get-AzMySqlConnectionString.ps1:37]: Done

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**